src/parsers/gas.ts

```diff
@@ -133,6 +118,35 @@ export const parseRainbowMeteorologyData = (
     confirmationTimeByPriorityFee,
     currentBaseFee,
   } = rainbowMeterologyData.data;
+
+  // API compatible
+  let blocksToConfirmation: BlocksToConfirmation;
+  if (!rainbowMeterologyData.data.blocksToConfirmationByBaseFee) {
+    const byPriorityFee: BlocksToConfirmationByPriorityFee = {
+      1: confirmationTimeByPriorityFee[15],
+      2: confirmationTimeByPriorityFee[30],
+      3: confirmationTimeByPriorityFee[45],
+      4: confirmationTimeByPriorityFee[60],
+    };
+    const byBaseFee: BlocksToConfirmationByBaseFee = {
+      120: new BigNumber(multiply(baseFeeSuggestion, 0.75)).toFixed(0),
+      240: new BigNumber(multiply(baseFeeSuggestion, 0.72)).toFixed(0),
+      4: new BigNumber(multiply(baseFeeSuggestion, 0.92)).toFixed(0),
+      40: new BigNumber(multiply(baseFeeSuggestion, 0.79)).toFixed(0),
+      8: new BigNumber(multiply(baseFeeSuggestion, 0.88)).toFixed(0),
```

Comment:
can we order these by increasing number and if lint complains, add comment to ignore error